{
  "vulinfo_id": "JVNDB-2017-013328",
  "title": "OpenSLP におけるバッファエラーの脆弱性",
  "vulinfo_description": "OpenSLP には、バッファエラーの脆弱性が存在します。",
  "affected": [
    {
      "name": "Debian",
      "product_name": "Debian GNU/Linux",
      "cpe": {
        "version": "2.2",
        "text": "cpe:/o:debian:debian_linux"
      }
    },
    {
      "name": "OpenSLP",
      "product_name": "OpenSLP",
      "version_number": [
        "1.0.2",
        "1.1.0"
      ],
      "cpe": {
        "version": "2.2",
        "text": "cpe:/a:openslp:openslp"
      }
    }
  ],
  "impact": {
    "cvss": [
      {
        "version": "2.0",
        "severity": {
          "type": "Base",
          "text": "High"
        },
        "base": "7.5",
        "vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
      },
      {
        "version": "3.0",
        "severity": {
          "type": "Base",
          "text": "Critical"
        },
        "base": "9.8",
        "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
      }
    ],
    "impact_item": "情報を取得される、情報を改ざんされる、およびサービス運用妨害 (DoS) 状態にされる可能性があります。"
  },
  "solution": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。",
  "related": [
    {
      "type": "vendor",
      "name": "Debian",
      "vulinfo_id": "[SECURITY] [DLA 1364-1] openslp-dfsg security update",
      "url": "https://lists.debian.org/debian-lts-announce/2018/04/msg00029.html"
    },
    {
      "type": "vendor",
      "name": "SourceForge",
      "vulinfo_id": "Lenovo fix for slpd crash during testing: bad pointer after realloc.",
      "url": "https://sourceforge.net/p/openslp/mercurial/ci/151f07745901cbdba6e00e4889561b4083250da1/"
    },
    {
      "type": "advisory",
      "name": "Common Vulnerabilities and Exposures (CVE)",
      "vulinfo_id": "CVE-2017-17833",
      "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-17833"
    },
    {
      "type": "advisory",
      "name": "National Vulnerability Database (NVD)",
      "vulinfo_id": "CVE-2017-17833",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2017-17833"
    },
    {
      "type": "cwe",
      "name": "JVNDB",
      "vulinfo_id": "CWE-119",
      "url": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
      "title": "バッファエラー"
    }
  ],
  "history": [
    {
      "history_no": "1",
      "date_time": "2018-06-21T05:37:50Z",
      "description": "[2018年06月21日]\\n  掲載"
    }
  ],
  "date_first_published": "2018-06-21T14:37:50+09:00",
  "date_last_updated": "2018-06-21T14:37:50+09:00",
  "date_public": "2017-12-22T00:00:00+09:00"
}
